### ISSUE TITLE
Unsupported Directory Import when using SSR

### ISSUE THREAD

**Issue by DanielHudson** [vue-icons-plus/vue-icons-plus#11]
When using this project with SSR with InertiaJS, we are receiving an error:

```
Error [ERR_UNSUPPORTED_DIR_IMPORT]: Directory import ''/Users/user/Herd/vue-project/node_modules/vue-icons-plus/lib' is not supported resolving ES modules imported from '/Users/user/Herd/vue-project/node_modules/vue-icons-plus/lu/index.mjs
```

Upon inspection it appears that the importing of the directories made via the following files is causing the issue:

https://github.com/vue-icons-plus/vue-icons-plus/blob/e4165eacf181efb914cb4fee44971cc494a28410/packages/core/scripts/task_icons.ts#L39

https://github.com/vue-icons-plus/vue-icons-plus/blob/e4165eacf181efb914cb4fee44971cc494a28410/packages/core/scripts/task_icons.ts#L47

https://github.com/vue-icons-plus/vue-icons-plus/blob/e4165eacf181efb914cb4fee44971cc494a28410/packages/core/scripts/task_all_files.ts#L55

Is it acceptable to adjust these to directly import the useGenIcon.mjs file ? I'd be happy to submit a PR for this, just not sure if it's acceptable.

Eg; 

`import { useGenIcon } from "../lib/useGenIcon.mjs";`

**Comment by dwanl** [comment]
> When using this project with SSR with InertiaJS, we are receiving an error:
> 
> ```
> Error [ERR_UNSUPPORTED_DIR_IMPORT]: Directory import ''/Users/user/Herd/vue-project/node_modules/vue-icons-plus/lib' is not supported resolving ES modules imported from '/Users/user/Herd/vue-project/node_modules/vue-icons-plus/lu/index.mjs
> ```
> 
> Upon inspection it appears that the importing of the directories made via the following files is causing the issue:
> 
> https://github.com/vue-icons-plus/vue-icons-plus/blob/e4165eacf181efb914cb4fee44971cc494a28410/packages/core/scripts/task_icons.ts#L39
> 
> https://github.com/vue-icons-plus/vue-icons-plus/blob/e4165eacf181efb914cb4fee44971cc494a28410/packages/core/scripts/task_icons.ts#L47
> 
> https://github.com/vue-icons-plus/vue-icons-plus/blob/e4165eacf181efb914cb4fee44971cc494a28410/packages/core/scripts/task_all_files.ts#L55
> 
> Is it acceptable to adjust these to directly import the useGenIcon.mjs file ? I'd be happy to submit a PR for this, just not sure if it's acceptable.
> 
> Eg;
> 
> `import { useGenIcon } from "../lib/useGenIcon.mjs";`

Of course, it's very welcome, but it could be changed to `import { useGenIcon } from “. /lib/index.mjs”;` and see if it works properly
